src/App.tsx

```diff
@@ -1,8 +1,8 @@
 import { BrowserRouter as Router, Route, Routes, Link } from 'react-router-dom';
 import LoginSignup from './pages/MainPage/Login';
 import FeedPage from './pages/MainPage/Main';
-import BookSearchPage from './pages/MainPage/BookSearch';
-import MyPage from './pages/MyPage/MyPage';
+import BookSearchPage from './pages/BookSearchPage/BookSearch';
+import MyPage from './pages/MainPage/Mypage';
```

Comment:
```suggestion
import MyPage from './pages/MyPage/MyPage';
```

이전 상태로 잘 못 돌았간 것 같습니다! 수정 부탁드려요!

Reply:
수정완료했습니다!